mygame.py

```diff
@@ -10,6 +10,7 @@
     'north', 'south', 'east', 'west',
     'n','s','e','w']
 fart_commands = ['fart']
+look_commands = ['look', 'search', 'investigate']
 quit_commands = ['quit']
 
 all_commands = info_commands + player_move_commands + fart_commands + quit_commands
```

Comment:
Let's also add look_commands to all_commands so the help command will show them.